lib/components/viewers/route-details.js

```diff
@@ -214,7 +217,15 @@ class RouteDetails extends Component {
                   route?.textColor
                 )}
               >
-                {stop.name}
+                <StopLink
+                  onClick={() => this._stopLinkClicked(stop.id)}
+                  textColor={getMostReadableTextColor(
```

Comment:
Agree the link needs better contrast. From talking to @demory I think color contrast is being tackled in other PRs? I also think `getMostReadableTextColor` only returns black and white as acceptable color options, so that may not be ideal for this case.

Reply:
I think black/white link is fine as long as it's underlined

Reply:
I meant to update the link color when I wrote that method but forgot!

Reply:
Okay! I'll go ahead and add it here, thanks!!